### ISSUE TITLE
Stops responding when logging in after "suspend"

### ISSUE THREAD

**Issue by MarwanAbdelatti** [Fmstrat/wintile#48]
The extension doesn't work when my laptop is back from "suspend" mode. Does not even let the default Ubuntu tiling work.
I had to disable and re-enable the extension to have it working

**Comment by gazhay** [comment]
I have a similar issue - it works if you drag a window to a hotzone, but keycombos stop working after sleep or suspend until you disable and re-enable the extension.

**Comment by Marwe** [comment]
Same here on Ubuntu 20.04 with gnome 1:3.30+2 and defined this alias to restart from cli:
```
alias wintile='gnome-extensions disable wintile@nowsci.com;gnome-extensions enable wintile@nowsci.com'
```

**Comment by rsxchin** [comment]
Resume after sleep/screenlock fixed in 4dec39dcc7f837e2bf57aa21008b462a5eef980e
